src/servises/formatString.ts

```diff
@@ -0,0 +1,3 @@
+export const formatString = (str: string): string => {
+  return str.split(' ').join(' ');
```

Comment:
The current implementation of `formatString` does not normalize spaces as intended. To replace multiple spaces with a single space, consider using a regular expression: `return str.replace(/\s+/g, ' ');`.